.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file shouldnt be in the repo. You can add it to the .gitignore if it's something you need on your system